src/pages/Animetoast/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Animetoast: pageInterface = {
+  name: 'Animetoast',
+  domain: 'https://www.animetoast.cc',
+  languages: ['German'],
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return $('.entry-title.light-title').text() != null && url.includes('link=');
+  },
+  isOverviewPage(url: string): boolean {
```

Comment:
Is true on pages like https://www.animetoast.cc/a-z-index/
Checking if the class .single-post exist is probably enough here.

Reply:
would you do it like this?
```ts
isSyncPage(url: string): boolean {
    return $('.single-post') != null && url.includes('link=');
  },
  isOverviewPage(url: string): boolean {
    return $('.single-post') != null && !url.includes('link=');
  },
  ```

Reply:
*pushed it allredy as change*